doc/manuals/contribution_guidelines.md

```diff
@@ -405,6 +405,17 @@ directly or indirectly are forced to have that construct as well.
 
 *How to check*: using an intelligent ```grep```.
 
+#### M12bis (`using namespace` in source code files)
+
+*Rule 1*: the construct ```using namespace XXX``` must not be used in source code files.
```

Comment:
We don't use "Rules 1", "Rule 2" style. As alternative, keep the "Rule", "Rationale", etc. subscection and use a bullet list with two items. This would make this rule homogenoeus among the others.